boards/common/stm32/include/cfg_usbdev_fs.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2021 Dylan Laduranty
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_common_stm32
+ * @{
+ *
+ * @file
+ * @brief       Common configuration for STM32 USB device FS peripheral
+ *
+ * @author      Dylan Laduranty <dylan.laduranty@mesotic.com>
+ */
+
+#ifndef CFG_USBDEV_FS_H
+#define CFG_USBDEV_FS_H
+
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Workaround inconsistency in STM32 vendor files */
+#ifndef USB_BASE
+#define USB_BASE USB1_BASE /**< USB Peripheral Base address */
+#endif /* ndef USB_BASE */
```

Comment:
I would put this workaround at CPU level.

Reply:
After comparing the CMSIS headers, on L4, the base address define is `USB_OTG_FS`. To me, it just seems that there's none on WB55. On L5, it's another story with NS/S addresses. On F1, it's simply `USB`.

Reply:
I think I should even remove it, both USB_BASE and USB1_BASE exists on WB but only USB_BASE exists for L4 IIRC but USB1_BASE (WB) == USB_BASE (L4) so this is too much confusing...

Reply:
Removed for now.